### ISSUE TITLE
Deprecation of Polymer

### ISSUE THREAD

**Issue by robinostlund** [kalkih/mini-media-player#752]
Hi,
Noticed today when i upgraded HA to 2023.4.0 that polymer is not going to be supported in 2023.5.0.
https://developers.home-assistant.io/blog/2023/04/04/deprecating_polymer/

And i saw that this repository uses that one. Any plans on replacing polymer with LIT? 😄

Brg
Robin

**Comment by breakthestatic** [comment]
I don't believe that this card actually uses Polymer at all. I'm guessing you might have done a search for the string `polymer` inside your www directory. I did so as well, but found that the only references to Polymer were all in licensing comments coming from the bundled lit dependencies. There didn't seem to be any actual references to the polymer library, and it's not a dependency of this project at all.

**Comment by Anto79-ops** [comment]
this is good to know. Indeed I used `Polymer.Element` and this integration came up. Good to know its just a licensing reference.
